src/components/control/select.jsx

```diff
@@ -180,75 +183,77 @@ const Select = forwardRef(function Select(props, ref) {
-          placeholder={props.placeholder}
-          readOnly={props.readOnly}
-          ref={refs.input}
-          required={props.required}
-          style={{ '--numIcon': 3 }}
-          type="text"
-          value={searchValue ?? defaultValue ?? ''}
-        />
-        <div className={styles['icon-container']}>
-          {validationMessage.length > 0 ? (<Icon
-            className={styles['input-icon']}
-            icon={cautionSvg}
-            label="Invalid custom field"
-          />) : undefined}
-          {!props.readOnly && (value || searchValue) ? (
-            <IconButton
-              icon={iconClear}
-              label={'Remove selected choice'}
-              onPress={clear}
-            />
-          ) : undefined}
+    <div className={classNames.container} style={{ position: 'relative' }}>
+      <input
+        autoComplete="off"
+        aria-autocomplete="none"
+        aria-controls={ids.listbox}
+        aria-haspopup="listbox"
+        aria-expanded={showOptions}
+        aria-invalid={invalid ? 'true' : undefined}
+        aria-describedby={`${ids.tooltip} ${ids.validation}`}
+        className={invalid ? classNames.invalidInput : classNames.input}
+        id={ids.input}
+        role="combobox"
+        name={props.name}
+        onBlur={onBlur}
+        onChange={onSearchChange}
+        onClick={onClick}
+        onKeyDown={onKeyDown}
+        onInput={props.onInput}
+        placeholder={props.placeholder}
+        readOnly={props.readOnly}
+        ref={refs.input}
+        required={props.required}
+        style={{ '--numIcon': 3 }}
+        type="text"
+        value={searchValue ?? defaultValue ?? ''}
+      />
+      <div className={styles['icon-container']}>
+        {validationMessage.length > 0 ? (<Tooltip
+          disabled={!props.validationMessageTooltip}
+          id=""
+          text={validationMessage}
+          direction="left"
+        >
           <Icon
             className={styles['input-icon']}
-            icon={props.readOnly ? iconCaretDownDisabled : iconCaretDown}
-            label={props.readOnly ? 'Select is not editable' : 'Open choices listbox'}
+            icon={cautionSvg}
+            id={ids.validation}
+            label={validationMessage}
           />
-        </div>
-        { showOptions && (
-          <Listbox
-            className={styles.dropdown}
-            id={`${props.id}-single-choice-listbox`}
-            labelledBy={`${props.id}-label`}
-            onChange={onSelectionChange}
-            ref={refs.listbox}
-            refs={props.listOptionRefs}
-            value={value}
-          >
-            {({ onSelect }) => {
-              const options = props.children({ onSelect });
-              if (React.Children.count(options) === 0) return <NoOptions />;
-              return options;
-            }}
-          </Listbox>
-        )}
+        </Tooltip>) : undefined}
+        {!props.readOnly && (value || searchValue) ? (
```

Comment:
It seems like this should be written:
```suggestion
        {!props.readOnly && ((value || searchValue) ? (
          ...
        ) : undefined)}
```

but maybe I have the big crazy.

Reply:
Good question. Pre-existing code tbf. 

Though, if you look at it, it is render JSX so we always want the return value of the whole thing to be JSX. 

```
a && (b || c) ? JSX : JSX

# can return either false, JSX, or JSX
```

```
a && ((b || c) ? JSX : JSX)

# can return either false, JSX, or JSX
```

I think we're good!